satpy/etc/composites/seviri.yaml

```diff
@@ -23,7 +23,7 @@ composites:
     standard_name: cloudtop
 
   cloudtop_daytime:
-    compositor: !!python/name:satpy.composites.GenericCompositor
+    compositor: !!python/name:satpy.composites.NotRecommendedCompositor
     prerequisites:
     - name: IR_039
       modifiers: [nir_emissive]
```

Comment:
`cloudtop_daytime` uses the `nir_emissive` modifier, doesn't this mean it shouldn't be deprecated without an equivalent being available?

Reply:
I guess you are right. It depends on how we choose to to use this "deprecation/not-recommended" flagging. I had thought we could warn users that they should stop using certain RGB's. This particular one is more of experimental character, and thus hardly used. And, we have really no experience with it. But still I wanted to have it included showing what you can do with Pyspectral. Maybe it is sufficient with a comment in the YAML file?

Reply:
I would say in the long run we should probably avoid "experimental" composites being in the main satpy package...maybe. Maybe the compositor needs to be `DeprecatedCompositor` (to emphasize that things are going away more than not being recommended) that includes a `description` parameter in the YAML that says what composite to use instead. Otherwise, maybe the description says why the composite is being deprecated (available in another package, experimental, etc). Thoughts?